code/modules/admin/admin.dm

```diff
@@ -367,7 +367,7 @@ GLOBAL_VAR_INIT(nologevent, 0)
 
 			if("Regular Restart")
 				var/delay = input("What delay should the restart have (in seconds)?", "Restart Delay", 5) as num|null
-				if(!delay)
+				if(!delay && delay != 0)
```

Comment:
This seems like an unrelated change.

Reply:
It doesn't require a new PR itself, it's a minor fix.